vcxagency-node/src/scripts/download-certs.js

```diff
@@ -0,0 +1,44 @@
+const { S3Client, GetObjectCommand } = require('@aws-sdk/client-s3')
+const fs = require('fs')
+const logger = require('../logging/logger-builder')(__filename)
+
+module.exports.fetchAwsAssets = async function fetchAwsAssets (appConfig) {
+  if (appConfig.SERVER_ENABLE_TLS === 'false') {
+    logger.info('TLS disabled, skipping AWS assets download...')
+    return
+  }
+
+  const certPath = appConfig.CERTIFICATE_PATH
+  const keyPath = appConfig.CERTIFICATE_KEY_PATH
+  const isCertAvailable = (appConfig.AWS_BUCKET_NAME && appConfig.AWS_DOMAIN_NAME) || (fs.existsSync(certPath) && fs.existsSync(keyPath))
```

Comment:
If `(fs.existsSync(certPath) && fs.existsSync(keyPath))` we are currently still trying to download the cert. I think should be:

```
if ((fs.existsSync(certPath) && fs.existsSync(keyPath)) {
   skip download; return;
}
if (appConfig.AWS_BUCKET_NAME && appConfig.AWS_DOMAIN_NAME) {
  download
} else {
   error, TLS enabled but no certs provided
}

Reply:
We are not trying to download them, but download only the missing one. You have a point that in a rare case when one file already exist but the other doesn't then we download the missing one, which is probably invalid and we should fail.